### DISCUSSION TITLE
How to add favorite product for each user?

### DISCUSSION THREAD

**Discussion by minhngvan** [saleor/saleor#10559] [Ideas]
I want to add a "favorite products" for each logged users. if I use metadata, not so good because too many metadata will be created (for each user)

**Comment by karolkielecki** [comment]
Hi @minhngvan, Saleor doesn't support it out of the box. I would say metadata is the only way.

**Reply by tiendq** [reply]
> Hi @minhngvan, Saleor doesn't support it out of the box. I would say metadata is the only way.

Is there any plan for wishlist/favorite feature?

**Reply by timuric** [reply]
> > Hi @minhngvan, Saleor doesn't support it out of the box. I would say metadata is the only way.
> 
> 
> 
> Is there any plan for wishlist/favorite feature?

any reason not to use metadata instead?

**Reply by timuric** [reply]
It should not be a problem if you have few hundreds of ids
